src/files-card-body.tsx

```diff
@@ -224,47 +233,125 @@ export const FilesCardBody = ({
+                        const newIdx = selectedIdx < sortedFiles.length - 1
+                            ? selectedIdx + 1
+                            : 0;
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "ArrowLeft":
+                if (hasNoKeydownModifiers(e)) {
+                    setSelected(_selected => {
+                        const firstSelectedName = _selected?.[0]?.name;
+                        const selectedIdx = sortedFiles?.findIndex(file => file.name === firstSelectedName);
+                        const newIdx = selectedIdx > 0
+                            ? selectedIdx - 1
+                            : sortedFiles.length - 1;
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "ArrowUp":
+                if (e.altKey && !e.shiftKey && !e.ctrlKey) {
+                    goUpOneDir();
+                } else if (hasNoKeydownModifiers(e)) {
+                    setSelected(_selected => {
+                        const firstSelectedName = _selected?.[0]?.name;
+                        const selectedIdx = sortedFiles?.findIndex(file => file.name === firstSelectedName);
+                        const newIdx = Math.max(selectedIdx - boxPerRow, 0);
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "ArrowDown":
+                if (e.altKey && !e.shiftKey && !e.ctrlKey && selected.length === 1) {
+                    onDoubleClickNavigate(selected[0]);
+                } else if (hasNoKeydownModifiers(e)) {
+                    setSelected(_selected => {
+                        const firstSelectedName = _selected?.[0]?.name;
+                        const selectedIdx = sortedFiles?.findIndex(file => file.name === firstSelectedName);
+                        const newIdx = Math.min(selectedIdx + boxPerRow, sortedFiles.length - 1);
+
+                        return [sortedFiles[newIdx]];
+                    });
+                }
+                break;
+
+            case "Enter":
+                if (hasNoKeydownModifiers(e) && selected.length === 1) {
+                    onDoubleClickNavigate(selected[0]);
+                }
+                break;
+
+            case "Delete":
+                if (hasNoKeydownModifiers(e) && selected.length !== 0) {
+                    confirm_delete(dialogs, currentPath, selected, setSelected);
+                }
+                break;
+
+            case "F2":
+                if (hasNoKeydownModifiers(e) && selected.length === 1) {
+                    show_rename_dialog(dialogs, path, selected[0]);
+                }
+                break;
+
+            case "a":
+                if (e.ctrlKey && !e.shiftKey && !e.altKey && !(e.target instanceof HTMLInputElement)) {
```

Comment:
It looks weird but it's a solution :)
I override default behavior for `ctrl + c`, `ctrl + v`, `ctrl + a` because it's used to select and copy/paste files. As a result it "breaks" default behavior of these keybinds in input fields such as the filed to filter current directory or the field when you start manually editing breadcrumbitem. So this way when the focused element is an input field and user presses `ctrl + a` it won't select all files in current directory but will select all text in the input field as expected.

Reply:
This sounds like it needs a comment as it isn't obvious enough. Something like this?

```tsx
// Keep standard text editing behavior by skipping inputs
```